docs/CostCenters.md

```diff
@@ -80,6 +84,29 @@ contxtSdk.facilities.costCenters
   .catch((err) => console.log(err));
 ```
 
+<a name="CostCenters+delete"></a>
+
+### costCenters.delete(costCenterId) ⇒ <code>Promise</code>
```

Comment:
Just curious but how do you do the arrow here? It's cool

Reply:
All of the `.md` files are created with rollup. Do an `npm run build` to generate the docs.